scheduler/duration_based_host_allocator_test.go

```diff
@@ -392,42 +390,6 @@ func TestOrderedScheduleNumNewHosts(t *testing.T) {
 	})
 }
 
-func TestSortDistrosByNumStaticHosts(t *testing.T) {
```

Comment:
How come this test is removed?

Reply:
I believe that this has also now been removed from master too, it tests code that was removed/invalidated by the fact that tasks can't specify more than one distro...